ffv1.md

```diff
@@ -288,6 +288,7 @@ SVGI:![svg](quantizationtablesets.svg "quantization table sets")
 SVGI:!---
 SVGC:quantizationtablesets.svg=$$Q_{j}[k]=quant\\_tables[i][j][k\\&255]$$
 AART:Q_(j)[k] = quant_tables[i][j][k&255]
+Figure: Description of the coding of the Quantized Sample Difference. {#figureQuantizationTableSets}
```

Comment:
This can be read as Quantized (Sample Difference) and (Quantized Sample) Difference, this should be worded so as to be free of this double interpretation

Reply:
"Quantized Sample Difference" is the wording used in the previous paragraph (actually already 6 times in the spec), should it be changed everywhere? It is a bit much as we are in the final part of the RFC, I am reluctant to change everywhere or use a wording not same as elsewhere.

Reply:
does each occurance of "Quantized Sample Difference" refer to the same thing ?

Reply:
> does each occurance of "Quantized Sample Difference" refer to the same thing ?

All but one are at the same place (in ["Quantization Table Sets" part](https://www.rfc-editor.org/authors/rfc9043.html#name-quantization-table-sets) or just after, in "Context"), the other one is in ["quant_tables" part](https://www.rfc-editor.org/authors/rfc9043.html#name-quant_tables) which refers to the same topic, as I understand.

Reply:
The quantization table converts sample differences in quantized (sample differences). But the storage of the quantization table has no samples, it stores really the quantization thresholds or levels or their differences. So iam a bit confused by the wording and what "Quantized Sample Difference" refers to here

Reply:
From my reading of Jerome's PR, I presumed that "Quantized Sample Difference" referred back to the expression of the same phrase earlier in the section.
> Quantization Tables are used on Sample Differences (see (#coding-of-the-sample-difference)), so Quantized Sample Differences are stored in the bitstream.

however, as the figure's label demonstrates how to compute the index (least 8 bits of the Quantized Sample Difference).

Rather than:
> Figure: Description of the coding of the Quantized Sample Difference. {#figureQuantizationTableSets}

how about
> Figure: Description of the coding of the index of the Quantized Sample Difference. {#figureQuantizationSampleDifferenceIndex}

Here the caption simply refers to the same terminology and assertions of the sentence that introduces the formula.

Reply:
I modified the patch accordingly.

Reply:
> From my reading of Jerome's PR, I presumed that "Quantized Sample Difference" referred back to the expression of the same phrase earlier in the section.
> 
> > Quantization Tables are used on Sample Differences (see (#coding-of-the-sample-difference)), so Quantized Sample Differences are stored in the bitstream.

Where are the Quantized Sample Differences stored ?

If i look at the text its not clearly worded
> Relative to any Sample `X`, the Quantized Sample Differences `L-l`, `l-tl`, `tl-t`, ` T-t`, and `t-tr` are used as context:
 `L-l`, `l-tl`, `tl-t`, ` T-t`, and `t-tr`  are Sample Differences, they become Quantized when passing through the quantization table, thats why its called a quantization table.

Maybe iam missing something but if not then the followig sounds more correct and maybe something like this could be used:
Figure: Description of the mapping from sample differences to the corresponding Quantized Sample Differences

Reply:
I updated @JeromeMartinez's patch per @michaelni's proposal.

Reply:
Acknowledging the @JeromeMartinez is afk but okays change with google hangouts discussion.

Reply:
Just retrieved my credentials :). Fine for me.